tests/unit/AlterationSummary.spec.ts

```diff
@@ -54,7 +54,6 @@ describe('Alteration Summary component', () => {
     expect(wrapper.find(AlterationSummary).exists()).toBe(true)
     expect(wrapper.find(EffectiveDateTime).exists()).toBe(true)
     expect(wrapper.find(ConfirmDialog).exists()).toBe(true)
-    expect(wrapper.find(NameTranslation).exists()).toBe(true)
```

Comment:
Actually, I think AlterationSummary _should_ test whether the NameTranslation sub-component is rendered (depending on various flags of course).

Reply:
Ok, I put some name translations in :)